### PR TITLE
[ISSUE #371]Translation for "Best Practice-消息轨迹" Section in the v4.x Document (CN -> EN)

### LINKED ISSUE #371: Translation for "Best Practice-消息轨迹" Section in the v4.x Document (CN -> EN)
The Chinese version of the "Best Practice-消息轨迹" section in the v4.x document has now been finished. However, its English version is still incomplete.

We now require someone who knows both Chinese and English and understands the technical terms of Apache RocketMQ to build it. The translation work is not only copy/paste from translation tools but also requires you to have some basic knowledge of Apache RocketMQ and technical writing.

### PR DESCRIPTION
**Pull request by mxsm**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

close #371 

## Brief changelog

- Translation for "Best Practice-消息轨迹" Section in the v4.x Document (CN -> EN)

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/17messagetra.md [hunk 1]
```diff
@@ -30,29 +30,29 @@ brokerIP1=XX.XX.XX.XX1
...
+By default, the message trace data is stored in the system-level TraceTopic (whose name is: **RMQ_SYS_TRACE_TOPIC**). This Topic is automatically created when the Broker node is started (as mentioned above, the **traceTopicEnable** switch variable must be set to **true** in the Broker side configuration file).
 
-### 3.2 用户自定义的TraceTopic 
-如果用户不准备将消息轨迹的数据存储于系统级的默认TraceTopic，也可以自己定义并创建用户级的Topic来保存轨迹（即为创建普通的Topic用于保存消息轨迹数据）。下面一节会介绍Client客户端的接口如何支持用户自定义的TraceTopic。
+### 3.2 User-defined TraceTopic
+If the user does not want to store the message trace data in the default system-level TraceTopic, they can also define and create a user-level Topic to save the trace (that is, create a regular Topic to store the message trace data). The next section will introduce how the Client interface supports user-defined TraceTopic.
 
-## 4. 支持消息轨迹的Client客户端实践
-为了尽可能地减少用户业务系统使用RocketMQ消息轨迹特性的改造工作量，作者在设计时候采用对原来接口增加一个开关参数(**enableMsgTrace**)来实现消息轨迹是否开启；并新增一个自定义参数(**customizedTraceTopic**)来实现用户存储消息轨迹数据至自己创建的用户级Topic。
+## 4. Supporting message trace Client practice
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/17messagetra.md]: Better change it to: The Client with Message Trace Practice

File: i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/17messagetra.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-| 发送消息时间 | 投递时间,投递轮次  | 消息存储位置 |
-| 消息是否发送成功 | 消息是否消费成功 | 消息的Key值 |
-| 发送耗时 | 消费耗时 | 消息的Tag值 |
+| Production instance information | Consumer instance information | Message Topic |
+| Time of message sending | Delivery time and delivery round | Message store location |
+| Success of message sending | Success of message consumption | Message key value |
+| Time taken to send | Time taken to consume | Message tag value |
 
-## 2. 支持消息轨迹集群部署
+## 2. Support for message trace cluster deployment
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/17messagetra.md]: How about "Cluster Deployment for Supporting Message Trace"?

File: i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/17messagetra.md [hunk 3]
```diff
@@ -88,30 +88,36 @@ RocketMQ的消息轨迹特性支持两种存储轨迹数据的方式：
...
-#Type      #ProducerGroup       #ClientHost          #SendTime            #CostTimes #Status
-Pub        1623305799667        xxx.xxx.xxx.xxx       2021-06-10 14:16:40  131ms      success
-```
+### 4.4 Use the mqadmin command to send and view traces
+- Send message
+
+  ```shell
+  ./mqadmin sendMessage -m true --topic some-topic-name -n 127.0.0.1:9876 -p "your meesgae content"
+  ```
+- Search trace
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/17messagetra.md]: Better change "Search" to "Query" here.

File: i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/17messagetra.md [hunk 4]
```diff
@@ -88,30 +88,36 @@ RocketMQ的消息轨迹特性支持两种存储轨迹数据的方式：
...
+
+  ```shell
+  ./mqadmin QueryMsgTraceById -n 127.0.0.1:9876 -i "some-message-id"
+  ```
+- Search trace result
```
Comment (tsunghanjacktsai) [inline i18n/en/docusaurus-plugin-content-docs/current/05-bestPractice/17messagetra.md]: Change "Search" to "Query" as well.

**Review by tsunghanjacktsai — CHANGES_REQUESTED** [review]
Hey @mxsm ,

Appreciate your contribution to our translation work. You've really done a great job on this. Only some minor word usage issues here, the rest looks fine on my side.

**Comment by mxsm** [comment]
@tsunghanjacktsai  thanks for you review, I will fix it later

**Comment by mxsm** [comment]
@tsunghanjacktsai  PTAL~

**Review by tsunghanjacktsai — APPROVED** [review]

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**